README.md

```diff
@@ -7,6 +7,12 @@
 crowbar makes it easy to write AWS Lambda functions in Rust. It wraps native Rust functions into CPython modules that
 handle converting Python objects into Rust objects and back again.
 
+## Alternatives
+
+As AWS Lambda has added more runtimes, more way to hack rust into the lambda have emerged.
+
+The [Rust on AWS Lambda](https://srijs.github.io/rust-aws-lambda/) project is kind enough to offer an alternative, and a comparison of itself to crowbar.
```

Comment:
I think the issues with that link can be fixed over there since the author of Rust on AWS Lambda seems happy enough to take reasonable changes... And in fact I think has already improved a bit since last time.

I think you're right that the bigger deal is the complexity of building the lambda in the first place, and if you'd rather me write a crowbar focussed snippet instead of linking to that one, I could.